dsc/configurations/write-compile-apply-configuration.md

```diff
@@ -42,6 +42,11 @@ Configuration HelloWorld {
 }
 ```
 
+> !Important
+> In more advanced scenarios where multiple modules need to be imported so you can work with many DSC Resources
+> in the same configuration, make sure to put each module in a seperate line using `Import-DscResource`.
```

Comment:
Could you add a code block here, showing the difference or an example of what you mean?

Reply:
trying code block within note.  if that doesn't work I will take it out of note annotation.